lib/cocoapods-core/source/manager.rb

```diff
@@ -394,8 +394,8 @@ def name_for_url(url)
         end
 
         case url.to_s.downcase
-        when %r{github.com[:/]+cocoapods/specs}
-          base = Pod::MasterSource::MASTER_REPO_NAME
+        when Pod::TrunkSource::TRUNK_REPO_URL.downcase
+          base = Pod::TrunkSource::TRUNK_REPO_NAME
```

Comment:
could this be updated to handle the trunk URL when it doesn't include the trailing `/`? I think we discussed this in Slack at some point

Reply:
+1 on trailing `/` I forgot about that.

Reply:
Fixed this.